spinalcordtoolbox/scripts/sct_check_dependencies.py

```diff
@@ -236,17 +238,43 @@ def main(argv: Sequence[str]):
     factor_MB = 1024 * 1024
     print('RAM: Total: {}MB, Used: {}MB, Available: {}MB'.format(ram.total // factor_MB, ram.used // factor_MB, ram.available // factor_MB))
 
+    if "+cpu" not in __torch_version__:
+        try:
+            status, n_gpus = run_proc([
+                "nvidia-smi",
+                "--query-gpu=count",
+                "--format=csv,noheader",
+                "-i", "0",
+            ], verbose=0, raise_exception=False)
+        except FileNotFoundError as e:
+            e.strerror = "GPU version of torch is installed, but could not find NVIDIA's GPU software"
+            raise e
```

Comment:
Thanks to macOS, I realize now that `+cpu` isn't solely enough of a check (during `sct_check_dependencies`).

macOS doesn't actually have distinct versions -- `torch` _is_ the CPU version, because there is no Mac GPU version of torch.